src/bit_shift.rs

```diff
@@ -0,0 +1,119 @@
+/// Defines shift operations
+
+#[repr(C)]
+pub struct BitwiseParam<T> {
+    pub input: T,
+    pub n: u32,
```

Comment:
in your ST signature, `n` asks for a DINT which is signed

Reply:
Good catch, I'll change the signature to `UDINT`
I don't think there's any way to work around `n` being bigger than the requested value however, the shift function will just crash then